web/src/features/llm-api-key/server/router.ts

```diff
@@ -41,6 +40,7 @@ export const llmApiKeyRouter = createTRPCRouter({
             baseURL: input.baseURL,
             withDefaultModels: input.withDefaultModels,
             customModels: input.customModels,
+            config: input.config,
```

Comment:
We are storing the secret key encrypted in the db above. In this case, the config contains secrets and is not stored encrypted.